app/dao/organisation_permissions_dao.py

```diff
@@ -0,0 +1,34 @@
+from app import db
+from app.models import OrganisationPermission
+
+# Default permissions for an organisation
+default_organisation_permissions = []
+
+
+def delete_organisation_permission(permission):
+    query = OrganisationPermission.query.filter_by(
+        permission=permission.permission, organisation_id=permission.organisation_id
+    )
+    query.delete()
+    db.session.commit()
+
+
+def set_organisation_permission(organisation, organisation_permissions, _commit=True, replace=True):
+    try:
+        if replace:
+            query = OrganisationPermission.query.filter_by(organisation=organisation)
+            query.delete()
+        for o_p in organisation_permissions:
+            o_p.organisation = organisation
+            db.session.add(o_p)
+    except Exception as e:
+        if _commit:
+            db.session.rollback()
+        raise e
+    else:
+        if _commit:
+            db.session.commit()
+
+
+def get_organisation_permissions_by_organisation_id(organisation_id):
+    return OrganisationPermission.query.filter_by(organisation_id=organisation_id).all()
```

Comment:
I don't think we need this - we should just be able to access `organisation.permissions` to get the same thing?